db/metamapper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Nextcloud - Notes
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ */
+
+namespace OCA\Notes\Db;
+
+use OCP\IDBConnection;
+use OCP\AppFramework\Db\Mapper;
+
+class MetaMapper extends Mapper {
+
+	public function __construct(IDBConnection $db) {
+		parent::__construct($db, 'notes_meta');
+	}
+
+	public function getAll($userId) {
+		$sql = 'SELECT * FROM `*PREFIX*notes_meta` WHERE user_id=?';
+		return $this->findEntities($sql, [$userId]);
+	}
+
+	public function get($userId, $fileId) {
+		$sql = 'SELECT * FROM `*PREFIX*notes_meta` WHERE user_id=? AND file_id=?';
+		return $this->findEntity($sql, [$userId, $fileId]);
+	}
```

Comment:
It would be nice, if there would be a way to do this. But I didn't find one (see https://github.com/nextcloud/notes/pull/51#issuecomment-279129420 for some use cases which have to be considered). If you have any idea on how to improve this, please give feedback. But also after the discussion with @BernhardPosselt, I don't see another way.